packages/obelisk/obelisk.0.5.0/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+version: "0.5.0"
+synopsis: "Pretty-printing for Menhir files"
+description: """
+Obelisk is a simple tool which produces pretty-printed output from a Menhir parser file (.mly).
+It is inspired from yacc2latex and is also written in OCaml, but is aimed at supporting features from Menhir instead of only those of ocamlyacc."""
+maintainer: ["Lélio Brun <lelio.brun@inria.fr>"]
+authors: ["Lélio Brun"]
+license: "MIT"
+homepage: "https://github.com/Lelio-Brun/Obelisk"
+doc: "https://github.com/Lelio-Brun/Obelisk"
+bug-reports: "https://github.com/Lelio-Brun/obelisk/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "2.2.0"}
+  "re"
+  "menhir"
+  "ocamlfind"
```

Comment:
I can't see any use of ocamlfind or findlib in your source code. Is this really needed?

I'm going to remove it as a test, if I'm wrong feel free to revert my change.